frappe/docs/user/en/guides/integration/google_gsuite.md

```diff
@@ -0,0 +1,68 @@
+# Google GSuite
+
+You can create and attach Google GSuite Docs to your Documents using your predefined GSuite Templates.
+These Templates could use variables from Doctype that will be automatically filled.
+
+## Enable integration with Google Gsuite
+
+### Publish Google apps script
+
+1. Go to [https://script.google.com](https://script.google.com)
+1. Create a new Project. Click on **File > New > Project**
+1. Fill Code.gs the content of ***<bench dir>/apps/frappe/integrations/doctype/gsuite_settings/gsuite_settings.gs***
```

Comment:
@rmehta 
you have 2 other options:
1. Publish a webapp a define as default in frappe (but google set limits that are very high for a single company but with a lot of user it could be a problem)
1. Use as a library (that google don't recommend because performance issues), but it is not that simplified that justify a change for a copy-paste.

please be sure I follow your concerns, but:
1. I've tried to install the webapp with REST, but google don't give chance to install, just run installed apps
1. Do you really think that an administrator of an ERP or a developer with frappe framework will not be able to follow a simple script of steps with a couple of copy paste? This isn't that different that activating google Social Login (create a project, copy credentials and past in frappe)...this integration just need to make a copy paste of a file before.